src/ActionPanel.scss

```diff
@@ -1,57 +1,91 @@
 @import 'variables';
 
-// TODO: Rework the arrow implementation. For example, it is not quite
-//       centered with respect to the action panel.
-
-.ActionPanel__panel {
-    width: $c2lc-action-panel-width;
+.ActionPanel__background {
+    align-self: center;
```

Comment:
align-self needs to be used in a flex or grid container; since this container is not one of them, this property has no effect. We can remove this line.

Reply:
Removed.